jakarta-package-alignment-mappings/src/main/java/com/palantir/gradle/jakartapackagealignment/VersionMapping.java

```diff
@@ -17,7 +17,7 @@
 
 import org.apache.maven.artifact.versioning.ComparableVersion;
```

Comment:
so one alternative to this is that we could leave `VersionMapping` package-private, and instead change the public `VersionMappings#getReplacement` to return an `Optional<MavenCoordinate>` instead of `Optional<String>`. From the perspective of something outside of this module, a caller doesn't care so much with inspecting the individual parts of `VersionMapping` and likely only cares about asking "what is the replacement that will be used for this dependency?"

Reply:
Sounds good to me, reducing the exposed surface is always a plus

Reply:
cool, fixed